### PR TITLE
Add a volume to keep bash history.

### PR DESCRIPTION
**Pull request by ValerioMagnago**
This PR should add a persistent volume to preserve the bash history. 
Let me know what you think ... for now I didn't had time to do a proper testing so there might be some problem hided in this PR.

### CODE REVIEW

File: .devcontainer/postCreateCommand.sh [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 
+# https://stackoverflow.com/questions/28279862/docker-and-bash-history
+echo "export HISTFILE=~/.persistent-volume/.bash_history" >> /home/ros/.bashrc
+export HISTFILE=~/.persistent-volume/.bash_history
+touch $HISTFILE
```
Comment (ValerioMagnago) [inline .devcontainer/postCreateCommand.sh:7]: HISTFILE is the environment variable storing the path to the bash history.

File: .vscode/settings.json [hunk 2]
```diff
@@ -69,28 +70,47 @@
         "*.xacro": "xml"
     },
     "search.exclude": {
```
Comment (ValerioMagnago) [inline .vscode/settings.json:72]: here I reoganize a bit the order of the stuff

File: .vscode/settings.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "python.analysis.inlayHints.functionReturnTypes": true,
     "python.analysis.inlayHints.variableTypes": true,
     // Use the system installed version of autopep8
+    "python.formatting.provider": "autopep8",
```
Comment (ValerioMagnago) [inline .vscode/settings.json:23]: here I added autopep8 as formatter is it ok?

File: .vscode/settings.json [hunk 4]
```diff
@@ -101,18 +121,9 @@
         "rosidl",
         "RTPS",
         "uncrustify",
+        "vesc",
```
Comment (ValerioMagnago) [inline .vscode/settings.json:124]: here I added some words that we often use but not in the dictionary

**Review by alessandroceleste — APPROVED** [review]
Made a fast test and it seems to work!

**Pull request merged**